README.md

```diff
@@ -98,7 +98,6 @@ docker run --rm -e S3_BUCKET=dlme-transform \
                 -e SKIP_FETCH_CONFIG=true \
```

Comment:
this line too?

Reply:
as always, an excellent :eye:! thank you. done.